packages/playground/tests/utils/isValidSmtp.test.ts

```diff
@@ -0,0 +1,24 @@
+import { describe, expect, it } from "vitest";
+
+import { isValidSmtp } from "../../src/utils/validators";
+
+describe("isValidSmtp", () => {
+  it("returns nothing for valid username and email", () => {
+    const input = "valid.username@example.com";
+    expect(isValidSmtp(input)).toBeUndefined();
+  });
+
+  it("returns an error message for special characters in username", () => {
+    const input = "invalid_email";
```

Comment:
`_` is not the only special character

Reply:
Of course not but I used one of them to test as I can't test all of them (too many cases).